src/app/space/create/deployments/services/deployments.service.spec.ts

```diff
@@ -1842,6 +1842,56 @@ describe('DeploymentsService', () => {
     });
   });
 
+  describe('#deleteDeployment', () => {
+    const spaceId = 'someSpaceId';
+    const environmentId = 'someStage';
+    const appId = 'someAppName';
```

Comment:
Could you move these const declarations inside the `it` block as well? I think it's acceptable that this means both `it` blocks will have duplicate declarations. And perhaps you can modify them so that their values will be modified after being run through `encodeURIComponent`, by adding a space character for example.